.bazelrc

```diff
@@ -1,9 +1,11 @@
-build --host_crosstool_top=@bazel_tools//tools/cpp:toolchain
-build --crosstool_top=//resim/toolchain:clang_suite
-build --cpu=k8
 build:debug -c dbg
 build:debug --copt="-g"
 build:debug --strip="never"
 
+common --enable_bzlmod
+
 coverage --java_runtime_version=remotejdk_11 --nocache_test_results
 coverage --combined_report=lcov --experimental_generate_llvm_lcov --experimental_use_llvm_covmap
+
+common --registry=https://raw.githubusercontent.com/resim-ai/open-core/prod/mikebauer/bazel-registry/bazel_registry/
```

Comment:
Is this right?

Reply:
It will be changed to point at main before landing, but otherwise yes.

Reply:
In fact, I can do that now that the other missing dep has landed. 🙂

Reply:
Done!